### PR TITLE
New package: Nlitesoftd.o.o.NTLite version 2024.4.9880.0

### PR DESCRIPTION
**Pull request by Jvr2022**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/149625)

### CODE REVIEW

**Comment by stephengillie** [comment]
Verify: 
image

File: manifests/n/Nlitesoftd/o/o/NTLite/2024.4.9880.0/Nlitesoftd.o.o.NTLite.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Nlitesoftd.o.o.NTLite
+PackageVersion: 2024.4.9880.0
+PackageLocale: en-US
+Publisher: Nlitesoft d.o.o.
+PackageName: NTLite
+License: none
+Copyright: © 2014-2024 Nlitesoft d.o.o.
+ShortDescription: NTLite Setup
```
Comment (stephengillie) [inline manifests/n/Nlitesoftd/o/o/NTLite/2024.4.9880.0/Nlitesoftd.o.o.NTLite.locale.en-US.yaml:11]: Hi @Jvr2022, 

Could the ShortDescription have more details? This line was taken from a guide for the software, so feel free to use a better description, if one is available. 

```suggestion
ShortDescription: NTLite is the best program for making an ISO for a Windows Environment.
```

Comment (Jvr2022) [reply manifests/n/Nlitesoftd/o/o/NTLite/2024.4.9880.0/Nlitesoftd.o.o.NTLite.locale.en-US.yaml:11]: I am gonna change it

**Comment by aisgbnok** [comment]
Is this different from:
https://github.com/microsoft/winget-pkgs/tree/master/manifests/n/Nlitesoft/NTLite/2024.4.9880.0 ?

**Pull request closed without merging**